src/main/scala/mesosphere/marathon/api/akkahttp/SystemController.scala

```diff
@@ -137,7 +115,10 @@ class SystemController(val marathonConfig: MarathonConf, val cfg: Config, val el
     // It would make sense to allow the functionality of this controller on every instance.
     asLeader(electionService) {
```

Comment:
Nit: Could you add `// format: OFF` here and `// format: On` after the routes so the alignment will not be screwed up by scala formatting?